docs/uml/macros.iuml

```diff
@@ -42,6 +42,39 @@ skinparam defaultMonospacedFontName Andale Mono
   !endif
 !endif
 
+!ifdef SHOW_DEPLOYMENT_HISTORY
+  !define SHOW_DEPLOYMENT
+
+  resource(DeploymentHistory) {
+    + <u>current</u> (\n\ttype: String,\n\tname: String,\n\trole: Maybe<Object>\n):\n\tMaybe<DeploymentHistory>
+    --
+    + type: String
+    + name: String
+    + role: Maybe<Object>
+    + {method} ref: Deployment
+    + previous: Maybe<DeploymentHistory>
+    ..
+    UNKNOWN_ID
+  }
+
+  DeploymentHistory o-- "1" Deployment
+  DeploymentHistory o-- "0..1" DeploymentHistory
+!endif
+
+!ifdef SHOW_DEPLOYMENT
+  !define SHOW_INSTANCE
+
+  resource(Deployment) {
+    + name: String
+    + role: Maybe<Object>
```

Comment:
What does the `role` object look like here?

Reply:
I haven't given this much thought yet